cnxarchive/database.py

```diff
@@ -60,19 +64,33 @@ def _read_sql_file(name):
 
 
 def initdb(settings):
-    """Initialize the database from the given settings."""
-    connection_string = settings[config.CONNECTION_STRING]
+    """Initialize the database from the given settings.
+       If a superuser is given in the settings then the
+       function will use the superuser to initialize the
+       database.
+    """
+    if settings['super-connection-string']:
+        connection_string = settings['super-connection-string']
+    else:
+        connection_string = settings['db-connection-string']
     with psycopg2.connect(connection_string) as db_connection:
         with db_connection.cursor() as cursor:
             for schema_filepath in DB_SCHEMA_FILE_PATHS:
                 with open(schema_filepath, 'r') as f:
                     cursor.execute(f.read())
+            # allow standard user to access the  'plpythonu' langauge.
+            cursor.execute(
+                "ALTER LANGUAGE plpythonu OWNER TO {user}".format(**settings))
```

Comment:
I think in that case, just add some comments so we know why it's done like that in the future. :)